### PR TITLE
Add how to use header

### PR DESCRIPTION
**Pull request by cbudz**
This PR adds a header to the introduction to better and more easily  inform readers of the order of operations on using the supplement.

### CODE REVIEW

**Comment by leswilliams44** [comment]
I approve the PR with the addition of the change to the introduction.

**Review by ndeevy — APPROVED** [review]

**Comment by ndeevy** [comment]
> Perhaps rewrite as "The Red Hat Customer Content Services team created this guide as part of its responsibility for producing product documentation for Red Hat customers. Upstream communities who want to align more closely with the standards used by Red Hat product documentation may also use the guide."

I like this rewrite, it's less passive.

**Comment by cbudz** [comment]
@leswilliams44 @ndeevy - I've added the recommended revision.

**Review by leswilliams44 — DISMISSED** [review]

File: supplementary_style_guide/introduction/about-guide.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/introduction/about-guide.adoc:10]: Typically there shouldn't be two level 1 headings in a file like this. I would make this a level 2 heading, or if you really think it needs to be a level 1, then move this section to another file.

**Review by bergerhoffer — COMMENTED** [review]
Just one minor thing!

**Comment by stoobie** [comment]
I just took a look at this chain. I'm on my phone, so adding my comment in
email.

Please don't use the word "may". Use "can" in this context. "May" is
ambiguous.

**Comment by bergerhoffer** [comment]
@cbudz Can you address the above feedback from @stoobie and I?

**Comment by gregraka** [comment]
@cbudz 

Just two minor changes from @bergerhoffer  and @stoobie and we can merge this PR!!

Would you be able to quickly knock this out?! :^)

We have the approvals!

Thanks,
--Greg

**Review by gregraka — APPROVED** [review]
Approved pending incorporation of Steve and Andrea's respective comments.

Thanks for you work on this, Chris (@cbudz)!

--Greg

**Comment by bergerhoffer** [comment]
@cbudz Do you mind updating this? Looks like just two minor tweaks then it can be merged.

**Review by mportman12 — APPROVED** [review]

**Comment by mportman12** [comment]
Made Andrea and Steve's changes and merged PR to keep the ball rolling.

**Comment by bergerhoffer** [comment]
Thanks @mportman12!

**Pull request merged**
